components/cli/ballerina/docker_bal_executor.go

```diff
@@ -137,12 +137,117 @@ func (baleExecutor *DockerBalExecutor) Build(fileName string, iName []byte) erro
 	return nil
 }
 
+// Run executes ballerina run when ballerina is not installed.
+func (balExecutor *DockerBalExecutor) Run(imageDir string, instanceName string, runningNode *DependencyTreeNode,
+	envVars []*EnvironmentVariable, tempRunFileName string, dependencyLinks map[string]*DependencyInfo,
+	startDependencies, shareDependencies bool) error {
+
+	currentDir, err := os.Getwd()
+	if err != nil {
+		util.ExitWithErrorMessage("Error in determining working directory", err)
+	}
+
+	//Retrieve the cellery cli docker instance status.
+	cmdDockerPs := exec.Command("docker", "ps", "--filter", "label=ballerina-runtime="+version.BuildVersion(),
+		"--filter", "label=currentDir="+currentDir, "--filter", "status=running", "--format", "{{.ID}}")
+
+	containerId, err := cmdDockerPs.Output()
+	if err != nil {
+		util.ExitWithErrorMessage("Docker Run Error", err)
```

Comment:
Same. Please check ExitWithErrorMessage in all places and return the error

Reply:
Updated in all places.